pkg/backup/compaction_job.go

```diff
@@ -702,7 +702,7 @@ func getBackupChain(
 	_, manifests, localityInfo, memReserved, err := backupdest.ResolveBackupManifests(
 		ctx, &mem, baseStores, incStores, mkStore, resolvedBaseDirs,
 		resolvedIncDirs, endTime, baseEncryptionInfo, kmsEnv,
-		user, false,
+		user, false /*includeSkipped */, true, /*includeCompacted */
```

Comment:
huh, strange that the linter isn't freaking out about the comma before `/*includeCompacted*/`. I feel like the comma should be after `/*includeCompacted*/`

Reply:
The linter is a paid actor.